package.json

```diff
@@ -44,7 +47,11 @@
   },
   "author": "",
   "babel": {
-    "presets": ["es2015", "stage-0", "react"],
+    "presets": [
+      "es2015",
+      "stage-0",
```

Comment:
@coopy @exogen I think we could cut it off at stage-2, since we're not using decorators but do need object rest spread...